README.md

```diff
@@ -175,10 +175,12 @@ One of the difficulties with CSS components is predictably. Unwanted styles
 can be inherited from parent components and this can make it difficult to
 reuse components in different contexts.
 
-Methodologies such as SUIT and BEM exist to try and mitigate this somewhat but
-what would really help is to allow inheritance to be 'opt-in' and let component
-authors to decide what properties are applied via inheritance. This creates a
-more predictable baseline for styling components and promoting easier reuse.
+Methodologies such as SUIT and BEM exist to solve problems around the cascade
+and specificity but they cannot protect components from inheriting unwanted
```

Comment:
Okay cool. I'll leave as is for now.

I sent the docs over to a friend who had no knowledge of this feature beforehand and he gave it a thumbs up in terms of making it easy to understand.  So that's good!